src/CraftRecaptcha.php

```diff
@@ -23,6 +24,10 @@
 use craft\events\RegisterUrlRulesEvent;
 use craft\contactform\models\Submission;
 
+use craft\elements\User;
+use \craft\services\Elements;
```

Comment:
I think you can drop the first \ here.

```suggestion
use craft\services\Elements;
```